docs/tutorials/bitcoin-price-feed/fine-tuning.md

```diff
@@ -17,9 +17,8 @@ const request = new Witnet.Request()
   .addSource(coindesk)       // Use source 2
   .setAggregator(aggregator) // Set the aggregation script
   .setTally(tally)           // Set the tally script
-  .setQuorum(4, 55)           // Set witness count
-  .setFees(1, 1)      // Set economic incentives
-  .schedule(0)               // Make this request immediately solvable
+  .setQuorum(100)            // Set witness count
+  .setFees(10, 1)            // Set economic incentives
```

Comment:
We can take this chance to set the bar higher, especially on collateralization.

Reply:
Good idea as the collateral / reward ratio is way off right now.